src/cli/commands/pack.js

```diff
@@ -6,6 +6,9 @@ import type {IgnoreFilter} from '../../util/filter.js';
 import * as fs from '../../util/fs.js';
 import {sortFilter, ignoreLinesToRegex, filterOverridenGitignores} from '../../util/filter.js';
 import {MessageError} from '../../errors.js';
+import * as util from 'util';
+import pipeline from 'stream.pipeline-shim';
```

Comment:
yarn's `engine` property in package.json claims node `>=4` which seems wrong.. but I'll investigate in the AM.